src/lib/components/ContentfulRichText/nodes/Node.svelte

```diff
@@ -7,19 +7,8 @@
   const component = nodes[node.nodeType];
 
   if (!component) {
-    // throw new Error(`unsupported nodeType ${node.nodeType}`);
+    throw new Error(`unsupported nodeType ${node.nodeType}`);
```

Comment:
Throwing an error here will cause a `500` yeah? Any chance we could log a warning instead and keep the if / else block below, rendering nothing if it's not supported? I'm still not sure if we've battened down the hatches on all the possible allowed rich text field references.

Reply:
good call, I'll move this to a `console.warn`